app/controllers/events_controller.rb

```diff
@@ -0,0 +1,70 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: %i[show edit update destroy]
+
+  # GET /events or /events.json
+  def index
+    @events = Event.all
+    @upcoming_invitations = Invitation.all.to_come
+    @past_invitations = Invitation.all.before_today
+  end
+
+  # GET /events/1 or /events/1.json
+  def show
+    @event = Event.find(params[:id])
+    @attendees = @event.attendees
+    # @current_inv = current_user.invitations.find_by(attended_event_id: @event.id)
+  end
+
+  # GET /events/new
+  def new
+    @event = current_user.created_events.new
+  end
+
+  # GET /events/1/edit
+  def edit; end
+
+  # POST /events or /events.json
+  def create
+    @event = current_user.created_events.invitations.build(event_params)
```

Comment:
- I got an error after tried to create a new post. Kindly work on it.
![Screen Shot 2021-08-01 at 10 26 24](https://user-images.githubusercontent.com/22258274/127776523-b303980f-7eaa-4724-adf8-19e164db5c9e.png)